appengine/firebase-event-proxy/gae-firebase-event-proxy/src/main/java/com/example/GaeFirebaseEventProxy/FirebaseEventProxy.java

```diff
@@ -73,7 +70,7 @@ public void onDataChange(DataSnapshot snapshot) {
         if (snapshot.exists()) {
           try {
             // Convert value to JSON using Jackson
-            String json = new ObjectMapper().writeValueAsString(snapshot.getValue());
+            String json = new ObjectMapper().writeValueAsString(snapshot.getValue(false));
 
             // Replace the URL with the url of your own listener app.
             URL dest = new URL("http://gae-firebase-listener-python.appspot.com/log");
```

Comment:
Ok for now, but it's not a good thing to ask users to change lines deep in the middle of the code.  My preferred place for GAE is to set it in the pom, so it can be overridden in the mvn command line -Dpyurl=... you could then update a properties file from mvn, using substitution ${pyurl} and read it in the code from the properties file.  If not that, then just set it as a constant early in the source file.